gems/smithy/spec/fixtures/protocol_tests/rpcv2_cbor/skipped_tests.smithy

```diff
@@ -0,0 +1,8 @@
+$version: "2.0"
+
+namespace smithy.protocoltests.rpcv2Cbor
+use smithy.ruby#skipTests
+
+apply OperationWithDefaults @skipTests([
+  { id: "RpcV2CborClientPopulatesDefaultValuesInInput", reason: "Defaults not Implemented yet.", type: "request" }
```

Comment:
Ohhhh nice, we just have a single test case that we need to resolve once defaults are implemented?

Reply:
I'm not sure - I think there are a few test cases - this was just a demo of how to use the trait/skip mechanism for now :-)